tests/graphql/errors/undeclared-type.graphql

```diff
@@ -0,0 +1,12 @@
+#> server-sdl
+schema @schema {
+  query: Query
+}
+
+type Query {
+  users: [User] @http(baseURL: "https://jsonplaceholder.typicode.com", path: "/users")
```

Comment:
```suggestion
  users0: [User] @http(baseURL: "https://jsonplaceholder.typicode.com", path: "/users")
  users1: [User] @http(path: "/users")
```
Ideally we should see two errors for `users1` field:
1. `The type 'User' was not found`.
2. `No base URL defined`.

Reply:
I've put this test case into separate test